data/json/items/gun/77mm_jap.json

```diff
@@ -0,0 +1,80 @@
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "type99",
+        "name": { "str_sp": "Arisaka Type 99" },
+        "description": "The standard service arm of the imperial Japanese army during the Second World War, chambered for the powerful 7.7x58mm cartridge and built to replace the weaker Type 38, the Type 99 is a sturdy and formidable bolt-action rifle with a Mouser-inspired action renowned for its strength.  Though the rifle was historically sneered at in the US due to extensive anti-Japanese propaganda and the lower quality of later-production guns, the weapon is, by most metrics, a finely produced and effective shoulder arm.  This particular Arisaka is an early-war example with anti-aircraft sights and an intact 6-pettle chrysanthemum seal.",
+        "weight": 1
+      },
+      {
+        "id": "type99ld",
+        "name": { "str_sp": "Arisaka Type 99" },
+        "description": "Standing as a clear example of what happens when a country losing a war realises that it’s running out of resources and its rifles are ripe for corner-cutting, this Type 99 Arisaka, the standard service arm of imperial Japan from 1939, is a crewed end-of-war example.  Coated in machining marks, lacking an upper handguard, and with the early production anti-aircraft sights long since done away with, where early production Arisaka rifles are a feat in elegant engineering, this one is a last-ditch bang stick hammered together in a mostly bombed-out factory.  Aesthetics not withstanding, this weapon is still an effective and powerful shoulder arm, just… not the prettiest."
+      }
+    ],
+    "weight": "3790 g",
+    "volume": "2680 ml",
+    "longest_side": "1118 mm",
+    "barrel_length": "657 mm",
+    "price": 65000,
+    "price_postapoc": 1850,
+    "to_hit": -1,
+    "material": [ "steel", "wood" ],
+    "color": "brown",
+    "ammo": [ "77mm_arisaka" ],
+    "dispersion": 150,
+    "durability": 10,
+    "blackpowder_tolerance": 24,
+    "clip_size": 5,
+    "flags": [ "RELOAD_ONE", "NEVER_JAMS", "EASY_CLEAN" ],
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "mechanism", 4 ],
+      [ "sights mount", 1 ],
+      [ "sling", 1 ],
+      [ "grip mount", 1 ],
+      [ "rail mount", 1 ],
+      [ "stock mount", 1 ],
+      [ "stock accessory", 2 ],
+      [ "underbarrel", 1 ]
+    ],
+    "//": "Having a single underbarrel slot precludes the mounting of both the monopod and the bayonet at the same time, but it's the only way to keep players from just stripping off the two and whacking on two underbarrel launchers or something.",
```

Comment:
Given the monopod is only usable on this rifle & folds up anyways, why not make it an integrated gunmod?

~~Puts on another monopod~~

Reply:
I didn’t make it a built-in or default gun mod as, according to what I’ve seen and read, the monopod is removable and is often missing from the rifles, as soldiers found that it wasn't particularly useful, and they were the first to get stripped from issue when material shortages knocked at the door.